### PR TITLE
vertexai[patch]: standardize model params

### PR DESCRIPTION
**Pull request by baskaryan**
Related to https://github.com/langchain-ai/langchain/issues/20085

### CODE REVIEW

**Comment by baskaryan** [comment]
mypy doesn't seem to understand the pydantic Field.alias + Config.allow_population_by_field_name typing  :/

File: libs/vertexai/tests/unit_tests/test_embeddings.py [hunk 1]
```diff
@@ -8,6 +8,16 @@
 from langchain_google_vertexai.embeddings import GoogleEmbeddingModelType
 
 
+def test_init() -> None:
+    for embeddings in [
+        VertexAIEmbeddings(
```
Comment (baskaryan) [inline libs/vertexai/tests/unit_tests/test_embeddings.py:13]: hm seems instantiating vertex embedding model requires setting up vertex creds which isn't great. is there a way to change that in a non-breaking fashion @lkuligin?

Comment (lkuligin) [reply libs/vertexai/tests/unit_tests/test_embeddings.py:13]: ideally it should be supported on the SDK side, I've raised a FR.

**Review by lkuligin — APPROVED** [review]

File: libs/vertexai/tests/unit_tests/test_embeddings.py (hunk 1 above)
Comment (lkuligin) [reply libs/vertexai/tests/unit_tests/test_embeddings.py:13]: so far I'd suggest let's move this to integration tests, wdyt?

Comment (baskaryan) [reply libs/vertexai/tests/unit_tests/test_embeddings.py]: moved 👍

**Pull request merged**